Makefile

```diff
@@ -1,6 +1,9 @@
 GOARCH := $(shell go env GOARCH)
 PWD=$(shell pwd)
 
+fmt: *.go
```

Comment:
This will only track the top-level go files, see:

```Makefile
demo: *.go
	ls -1 $^
```

```
~/Work/Code/envbuilder main* ⇣
❯ make demo
envbuilder.go
envbuilder_internal_test.go
envbuilder_test.go
git.go
git_test.go
log.go
log_test.go
options.go
options_test.go
```

vs

```
~/Work/Code/envbuilder main* ⇣ 6s
❯ ls -1 **/*.go
cmd/envbuilder/main.go
devcontainer/devcontainer.go
devcontainer/devcontainer_test.go
devcontainer/features/features.go
devcontainer/features/features_test.go
devcontainer/script.go
devcontainer/script_test.go
envbuilder.go
envbuilder_internal_test.go
envbuilder_test.go
git.go
git_test.go
integration/integration_test.go
log.go
log_test.go
options.go
options_test.go
scripts/docsgen/main.go
testutil/gittest/gittest.go
testutil/registrytest/registrytest.go
```